wagtail/wagtailembeds/format.py

```diff
@@ -2,36 +2,36 @@
 
 from django.template.loader import render_to_string
 
-from wagtail.wagtailembeds.embeds import get_embed
+from wagtail.wagtailembeds import embeds
 
 
 def embed_to_frontend_html(url):
     try:
-        embed = get_embed(url)
-        if embed is not None:
-            # Work out ratio
-            if embed.width and embed.height:
-                ratio = str(embed.height / embed.width * 100) + "%"
-            else:
-                ratio = "0"
-
-            # Render template
-            return render_to_string('wagtailembeds/embed_frontend.html', {
-                'embed': embed,
-                'ratio': ratio,
-            })
+        embed = embeds.get_embed(url)
+
+        # Work out ratio
+        if embed.width and embed.height:
+            ratio = str(embed.height / embed.width * 100) + "%"
         else:
-            return ''
-    except:
+            ratio = "0"
```

Comment:
I would set to `None` and then check in template rather than rendering `<div style="padding-bottom: 0;" class="responsive-object">` unless of course there is a default padding applied somewhere else that we're explicitly overriding here.